nx/source/runtime/devices/usb_comms.c

```diff
@@ -462,6 +462,29 @@ static Result _usbCommsRead(usbCommsInterface *interface, void* buffer, size_t s
     return rc;
 }
 
+static Result _usbCommsReadAsync(usbCommsInterface *interface, void *buffer, size_t size, u32 *urbId)
+{
+    if ((((u64)buffer) & 0xfff) || size > 0x1000) {
```

Comment:
This is an user-buffer not a tmpbuf, so this size check isn't needed (same with _usbCommsWriteAsync). Also: `(u64)` -> `(uintptr_t)`